OneginiExampleAppSwift/Interactors/MobileAuthInteractor.swift

```diff
@@ -22,73 +22,61 @@ protocol MobileAuthInteractorProtocol: AnyObject {
     func isUserEnrolledForMobileAuth() -> Bool
     func isUserEnrolledForPushMobileAuth() -> Bool
     func handlePinMobileAuth()
-    func fetchPendingTransactions(completion: @escaping (Array<ONGPendingMobileAuthRequest>?, AppError?) -> Void)
+    func fetchPendingTransactions(completion: @escaping (Array<PendingMobileAuthRequest>?, AppError?) -> Void)
     func handleMobileAuth()
     func handleCustomAuthenticatorMobileAuth()
-    func handlePendingMobileAuth(_ pendingTransaction: ONGPendingMobileAuthRequest)
+    func handlePendingMobileAuth(_ pendingTransaction: PendingMobileAuthRequest)
     func handleOTPMobileAuth(_ otp: String)
 }
 
 class MobileAuthInteractor: NSObject, MobileAuthInteractorProtocol {
     weak var mobileAuthPresenter: MobileAuthInteractorToPresenterProtocol?
     var mobileAuthQueue = MobileAuthQueue()
     var mobileAuthEntity = MobileAuthEntity()
-
-    override init() {
-        super.init()
-        UNUserNotificationCenter.current().delegate = self
+    private var userClient: UserClient {
+        UNUserNotificationCenter.current().delegate = self // ?
```

Comment:
that need to be moved somewhere, as we do not have initialisers

Reply:
Any ideas?

Reply:
as it was before?

`    override init() {
        super.init()
        UNUserNotificationCenter.current().delegate = self
    }`